src/main/java/pl/wojciechkostecki/revel/model/Local.java

```diff
@@ -21,5 +23,7 @@
 
     private LocalTime closingTime;
 
-    private String menu;
+    @OneToOne(cascade = CascadeType.ALL)
+    @JsonManagedReference
```

Comment:
Do you know what this annotation is doing?

Reply:
Yes, this annotation helped me eliminate the infinite recursion between bidirectionally linked entities. The above annotation is serialized. the back part of reference - @JsonBackReference - is ommited during serialization.